### PR TITLE
Clarify PR template

### PR DESCRIPTION
**Pull request by i-oden**
## Read this before submitting the PR

1. Always create a Draft PR first
2. Go through sections 1-5 below, fill them in and check all the boxes
3. Make sure that the branch is updated; if there's an "Update branch" button at the bottom of the PR, rebase or update branch.
4. When all boxes are checked, information is filled in, and the branch is updated: mark as Ready For Review and tag reviewers (top right)
5. Once there is a submitted review, implement the suggestions (if reasonable, otherwise discuss) and request an new review. 

If there is a field which you are unsure about, enter the edit mode of this description or go to the [PR template](../.github/pull_request_template.md); There are invisible comments providing descriptions which may be of help.

## 1. Description / Summary

This PR changes the PR template (copy pasted into this description) in order to simplify and clarify. 

## 2. Jira task / GitHub issue

DDS-1667

## 3. Type of change

What _type of change(s)_ does the PR contain?

**Check the relevant boxes below. For an explanation of the different sections, enter edit mode of this PR description template.**

- [ ] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [ ] Non-breaking 
- [ ] Database change: _Remember the to include a new migration version, **or** explain here why it's not needed._ 
- [ ] Bug fix 
- [ ] Security Alert fix 
- [ ] Documentation 
- [x] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [x] [Sprintlog](../SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [ ] PR to `master` branch: \_If checked, read [the release instructions](../doc/procedures/new_release.md) 
  - [ ] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](../.github/pull_request_template.md).**

- [ ] **Black**

- [ ] **Prettier**

- [ ] **Yamllint**

- [ ] **Tests**

- [ ] **CodeQL**

- [ ] **Trivy**

- [ ] **Snyk**

<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1467"></a>

### CODE REVIEW

**Comment by i-oden** [comment]
@valyo @rv0lt Checks are failing, and I haven't filled in the description, but you can review this now while I try and fix it anyway.  The PR changes the PR template to what is in the PR decription now.

File: dds_web/static/js/dds.js [hunk 1]
```diff
@@ -18,7 +18,7 @@
         }
         form.classList.add("was-validated");
       },
-      false,
+      false
```
Comment (i-oden) [inline dds_web/static/js/dds.js:21]:
```suggestion
      false,
```

File: dds_web/static/scss/_variables.scss [hunk 2]
```diff
@@ -28,19 +28,8 @@ $font-family-sans-serif:
...
-  Arial,
-  "Noto Sans",
-  "Liberation Sans",
-  sans-serif,
-  "Apple Color Emoji",
-  "Segoe UI Emoji",
-  "Segoe UI Symbol",
-  "Noto Color Emoji" !default;
+  -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans",
+  sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:32]:
```suggestion
  -apple-system,
  BlinkMacSystemFont, 
  "Segoe UI", 
  Roboto, 
  "Helvetica Neue", 
  Arial, 
  "Noto Sans", 
  "Liberation Sans",
  sans-serif, 
  "Apple Color Emoji", 
  "Segoe UI Emoji", 
  "Segoe UI Symbol", 
  "Noto Color Emoji" !default;
```

File: dds_web/static/scss/_variables.scss [hunk 3]
```diff
@@ -29,17 +29,17 @@ $font-family-sans-serif:
...
   system-ui,
   -apple-system,
-  BlinkMacSystemFont,
-  "Segoe UI",
-  Roboto,
-  "Helvetica Neue",
-  Arial,
-  "Noto Sans",
+  BlinkMacSystemFont, 
+  "Segoe UI",
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:33]:
```suggestion
  "Segoe UI",
```

File: dds_web/static/scss/_variables.scss [hunk 4]
```diff
@@ -29,17 +29,17 @@ $font-family-sans-serif:
...
   -apple-system,
-  BlinkMacSystemFont,
-  "Segoe UI",
-  Roboto,
-  "Helvetica Neue",
-  Arial,
-  "Noto Sans",
+  BlinkMacSystemFont, 
+  "Segoe UI", 
+  Roboto,
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:34]:
```suggestion
  Roboto,
```

File: dds_web/static/scss/_variables.scss [hunk 5]
```diff
@@ -29,17 +29,17 @@ $font-family-sans-serif:
...
-  "Segoe UI",
-  Roboto,
-  "Helvetica Neue",
-  Arial,
-  "Noto Sans",
+  BlinkMacSystemFont, 
+  "Segoe UI", 
+  Roboto, 
+  "Helvetica Neue", 
+  Arial,
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:36]:
```suggestion
  Arial,
```

File: dds_web/static/scss/_variables.scss [hunk 6]
```diff
@@ -29,17 +29,17 @@ $font-family-sans-serif:
...
+  Roboto, 
+  "Helvetica Neue", 
+  Arial, 
+  "Noto Sans", 
   "Liberation Sans",
-  sans-serif,
-  "Apple Color Emoji",
-  "Segoe UI Emoji",
-  "Segoe UI Symbol",
+  sans-serif,
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:39]:
```suggestion
  sans-serif,
```

File: dds_web/static/scss/_variables.scss [hunk 7]
```diff
@@ -29,17 +29,17 @@ $font-family-sans-serif:
...
+  "Noto Sans", 
   "Liberation Sans",
-  sans-serif,
-  "Apple Color Emoji",
-  "Segoe UI Emoji",
-  "Segoe UI Symbol",
+  sans-serif, 
+  "Apple Color Emoji", 
+  "Segoe UI Emoji", 
+  "Segoe UI Symbol",
```
Comment (i-oden) [inline dds_web/static/scss/_variables.scss:42]:
```suggestion
  "Apple Color Emoji",
  "Segoe UI Emoji",
  "Segoe UI Symbol",
```

**Review by rv0lt — APPROVED** [review]
seems good to me!

**Comment by i-oden** [comment]
> seems good to me!

@rv0lt Clear enough with the invisible comments etc?

**Comment by rv0lt** [comment]
> > seems good to me!
> 
> @rv0lt Clear enough with the invisible comments etc?

Yah, my only questions were about knowing when to check or uncheck a box. Now at least it seems clear to me :)

**Review by valyo — APPROVED** [review]

**Pull request merged**